src/components/sbb-button/sbb-button.tsx

```diff
@@ -1,27 +1,26 @@
 import { Component, Event, EventEmitter, h, Prop } from '@stencil/core';
 import { InterfaceButtonAttributes } from './sbb-button.custom';
 
-/**
- * @slot unnamed - Slot to render svg icon. You must pass an svg-element.
- */
-
 @Component({
   shadow: true,
   styleUrl: 'sbb-button.scss',
   tag: 'sbb-button',
 })
 export class SbbButton {
-  /** Label text to show on the button */
-  @Prop() public label? = '';
+  /** set as icon-only, no label, no text */
+  @Prop() public iconOnly = false;
```

Comment:
Is this property necessary? Could this not be inferred by checking whether nothing is assigned to the content slot?